Update CAM Google Cloud doc for instance protection, register and self-register (#2879)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Getting Started/register-existing-google-cloud-instance.md

### Audience

Users who want to register their *Google Cloud* instances into *Cloud Application Manager* to enable lifecycle management on them.

### Prerequisites

* A *Google Cloud* machine
* An active *Cloud Application Manager* account 
* An existing *Google Cloud* account with active instances

### Find a Google Cloud instance

Within the *Cloud Application Manager* workspace, navigate to your *Google Cloud* provider and run the sync function by clicking **Sync**.

The sync process will discover unregistered *Google Cloud* instances and list them under the **Unregistered Instances** tab.

![Locate Unregistered Instances](../../images/cloud-application-manager/register/cam-register-existing-google-cloud-instance-1.png)

### Register the instance

Under the **Unregistered Instances** tab or within the *Instances* page, click the instance you want to import and register.

Provide any details for the instance in the fields provided.

![Register Instance](../../images/cloud-application-manager/register/cam-register-existing-google-cloud-instance-2.png)

The automatic registration process will require reboot the virtual machines, or regenerated in case of an scaling group.

> To avoid that restarts, the *Install Agent Manually* togle can be enabled to install manually the CAM agent.

Reviewer: ```suggestion
To avoid the restart operation, the *Install Agent Manually* toggle can be enabled to install manually the Cloud Application Manager (CAM) agent.
```

Click **Register** to register the instance.

> If *Install Agent Manually* togle was enabled, then an error message is returned because the agent installation script that allows *Cloud Application Manager* to register the instance has not been executed on it yet. You can then proceed to the manual instalation process described below.

Reviewer: ```suggestion
If *Install Agent Manually* toggle was enabled, then an error message is returned because the agent installation script that allows *Cloud Application Manager* to register the instance has not been executed on it yet. You can then proceed to the manual installation process described below to fix the issue.
```

> ### Install the Agent Installation Script manually

Reviewer: ```suggestion
### Execute the Agent Installation Script manually
```

> Copy the agent installation script provided in the **Activity** tab in the log details.

Reviewer: ```suggestion
Copy the agent installation script provided in the **Activity** tab in the log details. This script is unique for each instance.
```

![Copy agent installation script](../../images/cloud-application-manager/register/cam-register-existing-google-cloud-instance-3.png)

Log-in to the instance residing in your *Google Cloud* provider.

Execute the agent installation script.

![Execute agent installation script](../../images/cloud-application-manager/register/cam-register-existing-google-cloud-instance-4.png)

> Once the script is executed, the instance will be available in *Cloud Application Manager*.

Reviewer: ```suggestion
Once the script is executed, the instance will be available in *Cloud Application Manager* as a registered instance.
```

![Cloud Application Manager Available](../../images/cloud-application-manager/register/cam-register-existing-google-cloud-instance-5.png)

> You can now use lifecycle application management features on this instance in *Cloud Application Manager*.

Reviewer: ```suggestion
Now you can use lifecycle application management features on this instance with *Cloud Application Manager*.
```

### Contacting Cloud Application Manager Support

We’re sorry you’re having an issue in [Cloud Application Manager](https://www.ctl.io/cloud-application-manager/). Please review the [troubleshooting tips](../Troubleshooting/troubleshooting-tips.md), or contact [Cloud Application Manager support](mailto:incident@CenturyLink.com) with details and screenshots where possible.

For issues related to API calls, send the request body along with details related to the issue.

In the case of a box error, share the box in the workspace that your organization and Cloud Application Manager can access and attach the logs.

* Linux: SSH and locate the log at /var/log/elasticbox/elasticbox-agent.log

> * Windows: RDP into the instance to locate the log at ProgramDataElasticBoxLogselasticbox-agent.log

Reviewer: ```suggestion
* Windows: RDP into the instance to locate the log at \ProgramData\ElasticBox\Logs\elasticbox-agent.log
```
